src/index.html

```diff
@@ -7,12 +7,42 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Calendar</title>
-    <link
-      rel="stylesheet"
-      href="styles/index.scss"
-    />
+    <link rel="stylesheet" href="styles/index.scss" />
```

Comment:
The link tag references a `.scss` file, which is not directly usable in browsers. Ensure that this SASS file is compiled to a CSS file and update the link to point to the resulting CSS file.